Content.Shared/Cloning/CloningPodComponent.cs

```diff
@@ -84,8 +132,17 @@ public enum CloningPodStatus : byte
     NoMind
 }
 
+[Serializable, NetSerializable]
+public enum ForcedMetempsychosisType : byte
+{
+    None,
+    Clone,
+    RandomHumanoid,
+    RandomNonHumanoid
+}
```

Comment:
Needed to do a Switch Case, which I could totally do via clamped integer(setting it >3 would just force NonHumanoid). The real reason the enum exists here is so that when an admin goes to set the CVar for forcing it, all 4 options will automagically appear in the CVar list, with the names. So that admins can just set the CVar to "RandomNonHumanoid" if they want only penguins and critters to come out of the machine. Doing it without the Enum meant I needed to have the logic for allowing forcing a specific metempsychosis be VERY UGLY.